### ISSUE TITLE
Accessibility issue - List Toggle buttons and Text area doesn't have Accessible name

### ISSUE THREAD

**Issue by dharanish-v** [mdx-editor/editor#459]
Facing issue trying to Access the Buttons in toolbar like specifically the List Toggle buttons, used the [axe devtools chrome plugin](https://www.deque.com/axe/devtools/chrome-browser-extension/) to check the accessibility of this. in addition to those three "list" buttons getting flagged, it turns out that the text input area also doesn't have an accessible name.

**Comment by petyosi** [comment]
You certainly know more about this than me - happy to accept a PR.

**Comment by dharanish-v** [comment]
sure, raised a PR
